iam/src/configure-iam.js

```diff
@@ -53,15 +54,17 @@ const configureIAM = async (
     // 3. Create DAS system (if not exists)
     promises.push(createNamespace(projectId, true, cluster, namespace)
       .then(() => checkSystem(systemName, styraToken, styraUrl)
-        .then((systemId) => {
-          if (systemId === '') {
+        .then((system) => {
+          if (system.id === '') {
             core.info(`creating system '${systemName}' in ${styraUrl}`);
             return setupSystem(
               namespace, systemName, env, repository, styraToken, styraUrl, systemOwners,
             );
           }
           core.info(`system '${systemName}' already exists in ${styraUrl}`);
-          return checkOwners(systemId, styraToken, styraUrl, systemOwners);
+          promises.push(checkOwners(system.id, styraToken, styraUrl, systemOwners));
+          promises.push(checkRepository(system, styraToken, styraUrl, repository));
```

Comment:
Is it OK that ´checkOwners` and `checkRepository` run concurrently, potentially both updating the DAS system? The alternative would be to chain them to run after one another.